Signal/src/call/WebRTCCallMessageHandler.swift

```diff
@@ -29,23 +29,20 @@ class WebRTCCallMessageHandler: NSObject, OWSCallMessageHandler {
 
     public func receivedOffer(_ offer: OWSSignalServiceProtosCallMessageOffer, from callerId: String) {
         AssertIsOnMainThread()
-        Logger.verbose("\(TAG) handling offer from caller:\(callerId)")
```

Comment:
These log entries are redundant with what's already in CallService.

e.g. it's not that helpful to have logs like this:
```
[CallHandler] Called Foo
[CallService] Called Foo
[CallHandler] Called Bar
[CallService] Called Bar
```

Let's just log in CallService only.